mapsims/cmb.py

```diff
@@ -0,0 +1,81 @@
+import healpy as hp, numpy as np
+from pixell import enmap, curvedsky
+
+def get_cmb_sky(iteration_num, 
+                nside = None, #set this if healpix is desired
```

Comment:
please format the code automatically with `black` https://github.com/ambv/black

Reply:
Done.